api-client/package.json

```diff
@@ -16,8 +16,8 @@
     "eslint": "eslint ."
   },
   "dependencies": {
-    "@babel/polyfill": "^7.8.3",
-    "@babel/runtime": "^7.8.3",
+    "@babel/polyfill": "^7.12.1",
```

Comment:
Caret matching for dependencies and exact version for dev dependencies

Reply:
Probably its fine to use different babel packages with different versions, right? Because we now have e.g. `7.12.1`, `7.12.10`, ...

Reply:
yeah this is something which you do not see when using caret versions.
Some libraries are not yet updates in babel